week3/Borislav/issue_TAPC-3.py

```diff
@@ -0,0 +1,46 @@
+# -*-config: utf_8 -*-
```

Comment:
Please rename the filenames so are python compliant.

Reply:
I don't understand. You want to rename the files, for example : docstring_validator.py. This is correct or is something else.
Can you explain in detail.

Reply:
Yes, please rename the files so they don't contain `-` or special characters that prevents from importing. Use snake case.